src/python/pants/option/global_options.py

```diff
@@ -486,15 +486,12 @@ def register_bootstrap_options(cls, register):
             help="Write logs to files under this directory.",
         )
 
-        # This facilitates bootstrap-time configuration of pantsd usage such that we can
-        # determine whether or not to use the Pailgun client to invoke a given pants run
-        # without resorting to heavier options parsing.
         register(
             "--enable-pantsd",
             advanced=True,
             type=bool,
-            default=False,
-            help="Enables use of the pants daemon (and implicitly, the v2 engine). (Beta)",
+            default=True,
```

Comment:
Technically, per the deprecation policy, changing a default value should only happen after a deprecation cycle. https://pants.readme.io/docs/deprecation-policy#prohibited-changes

We can update the docs so that new users are configuring this to be true, though. And I think we've agreed we want more frequent and smaller stable releases, so we will hopefully get to 1.31.0 sooner.